homeassistant/components/dublin_bus_transport/sensor.py

```diff
@@ -3,9 +3,6 @@
 
 For more info on the API see :
 https://data.gov.ie/dataset/real-time-passenger-information-rtpi-for-dublin-bus-bus-eireann-luas-and-irish-rail/resource/4b9f2c4f-6bf5-4958-a43a-f12dab04cf61
```

Comment:
Not sure if this can or should be removed as well.

Reply:
That can stay.